e_qat.c

```diff
@@ -516,6 +520,7 @@ int qat_engine_init(ENGINE *e)
     if (CPA_STATUS_SUCCESS != status) {
         WARN("cpaCyGetInstances failed, status=%d\n", status);
         QATerr(QAT_F_QAT_ENGINE_INIT, QAT_R_GET_INSTANCE_FAILURE);
+        pthread_key_delete(thread_local_variables);
```

Comment:
This came out from in our internal review and missed it to put the comment here.  Suggestion is to place pthread_key_delete in qat_engine_finish_int() that gets called from qat_engine_finish()? Then you would only need to add it once rather than in all these places. It would still need to be called at line 479 though. Is that fine we change that locally and merge that change which should be available after the release ?

Reply:
ok, thanks.

Reply:
When you merge change，can you add: 
Signed-off-by: Yunfeng Ye <yeyunfeng@huawei.com>

thanks.